src/ReactiveUI/Routing/RoutingState.cs

```diff
@@ -13,6 +13,7 @@
 using System.Runtime.Serialization;
 using DynamicData;
 using DynamicData.Binding;
+#pragma warning disable 8618
```

Comment:
Remove `SetupRX()` and put it in the constructor, suspect that will make your nullables work without the need for this pragma.